.gitignore

```diff
@@ -2,3 +2,4 @@
 barys.log
 build/node_modules
 *.log
+build/package-lock.json
```

Comment:
I don't have this file generated in any of my repositories. @agherzan @ZubairLK do you get this created?